### ISSUE TITLE
Cannot add a new value to the INI for Black Ops Cold War

### ISSUE THREAD

**Issue by 14142000** [dafzor/bnetlauncher#47]
I added a new value at the bottom the file for Cold War and it wont launch 
![Screenshot 2020-10-17 113647](https://user-images.githubusercontent.com/3375830/96348264-0ec1e580-106d-11eb-9c7d-b389ffb25d51.png)

![Screenshot 2020-10-17 113759](https://user-images.githubusercontent.com/3375830/96348297-429d0b00-106d-11eb-8175-67f371bf387a.png)

**Comment by dafzor** [comment]
command codes are case sensitive and in this case it's not "zeusb", bellow is the corrected entry:

```
[codbocw]
name=Call of Duty: Back Ops Cold War
client=battlenet
cmd=ZEUS
exe=BlackOpsColdWar.exe
options=waitforexit
```

**Comment by dafzor** [comment]
Fixed with release 2.11
